optaplanner-website-root/content/blog/2022/09/26/DynamicScalingOfOptaPlannerWorkloadOnOpenShift.adoc

```diff
@@ -0,0 +1,91 @@
+= Run OptaPlanner workloads on OpenShift, part II.
+rsynek
+2022-09-26
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: cloud, openshift, kubernetes, scaling
+
+In the first https://www.optaplanner.org/blog/2022/06/09/RunOptaPlannerWorkloadOnOpenShift.html[article about running OptaPlanner workloads on OpenShift],
+I've shown how to use the OptaPlanner operator to solve multiple datasets of the same planning problem by starting several pods on OpenShift.
+
+Let's not pay the bills for machines sitting idle. Since the previous article, the OptaPlanner operator supports dynamic scaling driven by the number of input datasets.
```

Comment:
```suggestion
That implementation kept a number of nodes always around, even if they were idle. But why pay the bills for those? Since the previous article, the OptaPlanner operator supports dynamic scaling driven by the number of input datasets.
```